app/controllers/spree/admin/images_controller.rb

```diff
@@ -50,19 +50,20 @@ def update
         @url_filters = ::ProductFilters.new.extract(request.query_parameters)
         set_viewable
 
-        if @object.update(permitted_resource_params)
+        if @object.update!(permitted_resource_params)
```

Comment:
I respect this approach, to consider invalid records as an exception. However it's not the usual approach so I would avoid it unless there was an added advantage in this case.